src/components/content/order/common/FlavorSelection.tsx

```diff
@@ -51,7 +111,47 @@ export const FlavorSelection = ({
                                             className={flavorStyles.customRadioButtonContent}
                                         >
                                             {FlavorTitle(flavor.name)}
-                                            <div className={flavorStyles.flavorPriceContent}>{FlavorPrice()}</div>
+                                            {getServicePriceQuery.isSuccess ? (
+                                                <div className={flavorStyles.flavorPriceContent}>
+                                                    {FlavorPrice(flavor, getFlavorWithPricesList())}
+                                                </div>
+                                            ) : getServicePriceQuery.isError ? (
```

Comment:
not just the API call error. But also error inside the FlavorPriceResult object must be handled.